src/styles/blocks/product.scss

```diff
@@ -0,0 +1,21 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__photo {
+    width: 100%;
+    border-radius: 4px;
+  }
+
+  &__price {
+    bottom: 0;
```

Comment:
The 'bottom: 0;' property for the '.product__price' class is unnecessary as it does not affect the layout. Consider removing it.